src/style.css

```diff
@@ -1,3 +1,110 @@
+html {
+  --header-height: 60px;
+  --active-link-color: rgba(0, 172, 220, 1);
+  --main-text-color: black;
```

Comment:
Good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector:
```css
:root {
  --header-height: 60px;
  --active-link-color: rgba(0, 172, 220, 1);
  --main-text-color: black;
}
```

Reply:
+